app_flutter/lib/status_grid.dart

```diff
@@ -24,14 +25,16 @@ class StatusGrid extends StatelessWidget {
     return SingleChildScrollView(
       scrollDirection: Axis.horizontal,
       child: Container(
-        width: taskCount * 50.0,
-        height: 500.0,
-        child: GridView.count(
+        width: columnCount * 50.0,
+        height: MediaQuery.of(context).size.height - 56,
```

Comment:
I think you should be able to get the constrain system to lay this our properly with hard coding any numbers here. What happens if you place (in the parent widget, a column with a fix sized container? Like:
```
Column(children: [
  SizedBox(height: 56),
  <<ThisWidget>>
]);
```

Or does it need a specific size to work?

Reply:
It needed a specific size to work.

Implementing the SizedBox just added 56 pixels to the overflow. Are there any other layout widgets I should try, as I don't like the idea of changing this magic number to prevent overflow.

Reply:
I think the solution will likely involve a Viewport of some kind. That would essentially remove the overflow problem by allowing the "inside to be bigger", so to speak

Reply:
Expanded did the trick, and made it so the Grid didn't need to explicitly define height

Reply:
No overflow issues?